docs/devices-api/more-info/webhooks.mdx

```diff
@@ -17,15 +16,79 @@ how the Webhook flow works, see [our Webhooks guide](/home/api-info/webhooks).
   page](/devices-api/more-info/our-integrations).
 </Info>
 
-## User data message
+## User provider connection message
+
+Example payload:
+
+```json
+{
+  "meta": {
+    "messageId": "<message-id>",
+    "when": "<date-time-in-utc>",
+    "type": "devices.provider-connected"
+  },
+  "users": [
+    {
+      "userId": "951faef1-5cfd-464a-81f7-31f76edf309e",
+      "provider": "garmin",
+      "connectedProviders": ["fitbit", "garmin", "cronometer"]
+    }
+  ]
+}
+```
 
 The format follows:
 
+<ResponseField name="meta" type="string[]" required>
+  Metadata about the message.
+  <Expandable title="meta properties">
+
+    <ResponseField name="messageId" type="string" required>
+      The ID for this message from Metriport, useful for debugging purposes only;
+    </ResponseField>
+
+    <ResponseField name="when" type="string" required>
+      The timestamp when this message was originally sent, formatted as [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601)
+      (example: `2022-12-24T00:46:05.413Z`)
+    </ResponseField>
+
+    <ResponseField name="type" type="string" required>
+      The type of the webhook message.
+    </ResponseField>
+
+  </Expandable>
+</ResponseField>
```

Comment:
I suggest we move this out of the "message" section so we don't repeat it over and over, as well as have a specific section about the metadata.

Maybe add this section on the overall Webhook area? https://docs.metriport.com/home/api-info/webhooks